src/Symfony/Component/DependencyInjection/Compiler/AutowirePass.php

```diff
@@ -271,6 +273,12 @@ private function autowireMethod(\ReflectionFunctionAbstract $reflectionMethod, a
 
                         break;
                     }
+
+                    if (Inner::class === $attribute->getName()) {
+                        $arguments[$index] = new Reference( "$class.inner", ContainerInterface::NULL_ON_INVALID_REFERENCE);
```

Comment:
just `.inner` should work better

Reply:
hum actually not since DecoratorServicePass ran before
but then, this should be `$this->currentId.'.inner'`